druid-shell/src/platform/windows/window.rs

```diff
@@ -1075,8 +1079,20 @@ impl WindowHandle {
     // TODO: Implement this
     pub fn show_titlebar(&self, _show_titlebar: bool) {}
 
-    // TODO: Implement this
-    pub fn resizable(&self, _resizable: bool) {}
+    pub fn resizable(&self, resizable: bool) {
+        if let Some(w) = self.state.upgrade() {
+            let hwnd = w.hwnd.get();
+            unsafe {
+                let mut style = GetWindowLongPtrW(hwnd, GWL_STYLE);
+                if resizable {
+                    style |= (WS_THICKFRAME | WS_MAXIMIZEBOX) as LONG_PTR;
+                } else {
+                    style &= !(WS_THICKFRAME | WS_MAXIMIZEBOX) as LONG_PTR;
```

Comment:
Need to use `WindowLongPtr` instead of `LONG_PTR` for 32-bit support.

Reply:
Is `WindowLongPtr` a type? I can't seem to find it in the winapi docs. From what I understand `GetWindowLongPtrW` and `SetWindowLongPtrW` returns and uses `LONG_PTR` which is 32 bit on 32 bit windows, and 64 bit on 64 bit windows.

Reply:
It's a bit confusing for sure.

1. `WindowLongPtr` is a druid-shell type defined in `window.rs` aka this very same file. You can search for it to see its definition.
2. `GetWindowLongPtrW` only works on 64-bit but the Rust `winapi` crate we're using correctly aliases that function to `GetWindowLongW` in 32-bit mode.

Reply:
Ah, thanks! I didn't think to look in druid for `WindowLongPtr` type.. ;) I'm a bit confused though as to why it is neccesary: 

- According to MSDN `LONG_PTR` is defined to be the size of a pointer, that is 4 bytes on 32 bit windows, and 8 bytes on 64 bit windows. [LONG_PTR docs](https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types)
- In the winapi crate `LONG_PTR` is defined to be `isize`, which also is 4 bytes on 32 bit windows, and 8 bytes on 64 bit windows: [isize docs](https://doc.rust-lang.org/std/primitive.isize.html), [winapi LONG_PTR docs](https://docs.rs/winapi/0.3.8/winapi/shared/basetsd/type.LONG_PTR.html) 

Am I missing something?

Reply:
Ah I found issue #437, I had expected this to be the same between using the windows API in C++ and rust.. Will change it :)

Reply:
Well the docs for [GetWindowLongPtrW](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getwindowlongptrw) state that for 32-bit support you should use `GetWindowLongPtr` instead which is aliased to `GetWindowLong` on 32-bit.

The rust `winapi` crate aliases `GetWindowLongPtrW` to `GetWindowLongW` on 32-bit. The `GetWindowLongW` expects `i32` while `GetWindowLongPtrW` expects `isize`.

Reply:
Alright, makes sense. [Seems like it's something that might be getting changed in winapi 0.4](https://github.com/retep998/winapi-rs/pull/735). Thanks for explaining!